drivers/interrupt_controller/Kconfig.multilevel.aggregator_template

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
-config $(cur-level)_LVL_INTR_0$(aggregator)_OFFSET
+config $(cur-level)_LVL_INTR_$(aggregator)_OFFSET
```

Comment:
Don't the 3rd level `aggregator = 0`, `aggregator = 1`, etc. lines in drivers/interrupt_controller/Kconfig.multilevel need to be modified to take this change into account as well? It looks like this patch will break 3rd level IRQ handlers as-is.